pkg/auth/oauth/handlers/default_auth_handler.go

```diff
@@ -146,6 +147,11 @@ func (authHandler *unionAuthenticationHandler) AuthenticationNeeded(apiClient au
 	}
 
 	if (len(authHandler.redirectors)) == 1 {
+		// TODO JWF should be if the config setting is set, redirectors should really be an array for consistent ordering
+		if true {
+			authHandler.selectProvider.render.Render(authHandler.redirectors, w, req)
```

Comment:
@liggitt we need to be passing down all the additional config data we talked about at this point, but the type passed  into the default auth handler is the AuthenticationRedirector interface type, would you add a metadata function to the interface that returns a struct type AuthenticationRedirectorMeta (or whatever its called).  other suggestions?